pyannote/database/util.py

```diff
@@ -382,6 +382,31 @@ def load_uem(file_uem):
 
     return timelines
 
+def load_lab(path, uri: str = None) -> Annotation:
+    """Load LAB file
+
+    Parameter
+    ---------
+    file_lab : `str`
+        Path to LAB file
+        (e.g. "/path/to/{uri}.lab")
```

Comment:
```suggestion
```

This `path` is supposed to contain an actual path, not a path with a placeholder.
The example between brackets can therefore be removed.